cmplibrary/src/main/java/com/sourcepoint/cmplibrary/consent/ConsentManager.kt

```diff
@@ -84,8 +84,10 @@ private class ConsentManagerImpl(
     override var sPConsentsError: ((Throwable) -> Unit)? = null
 
     override val storedConsent: Boolean
```

Comment:
I believe that the names of the boolean flags should start by `is...`, `has...`, etc

so in this case it is better to name it `hasStoredConsent`, which will indicate whether or not the `DataStorage` contains either gdpr or ccpa consents

Reply:
but this is just a suggestion for now

Reply:
Makes sense